frontend/src/routes/+layout.svelte

```diff
@@ -2,21 +2,50 @@
   import "../app.scss";
 </script>
 
-<div class="columns m-6">
-  <div class="column is-one-third" />
-  <div class="column is-one-third">
-    <div class="level">
-      <div class="level-item">
-        <h1 class="title is-0">Rational Trading</h1>
-      </div>
+<!-- the navbar -->
+<nav class="navbar is-primary">
+  <div class="navbar-brand">
+    <div class="navbar-item">
+      <img src="/logo.png" width="177" height="50" alt="logo" />
```

Comment:
Maybe make the logo a clickable link to homepage?

Reply:
Sure